src/main/java/io/vertx/ext/mongo/AggregateOptions.java

```diff
@@ -25,20 +25,19 @@
    */
   public static final long DEFAULT_MAX_AWAIT_TIME = 1000L;
 
-  private int     batchSize;
-  private long    maxTime;
-  private long    maxAwaitTime;
+  private int batchSize;
+  private long maxTime;
+  private final long maxAwaitTime;
   private Boolean allowDiskUse;
-  private CollationOptions collationOptions;
-
+  private CollationOptions collation;
   /**
    * Default constructor
    */
   public AggregateOptions() {
     this.batchSize = DEFAULT_BATCH_SIZE;
     this.maxTime = DEFAULT_MAX_TIME;
     this.maxAwaitTime = DEFAULT_MAX_AWAIT_TIME;
-    this.collationOptions = null;
+    this.collation = null;
```

Comment:
You can't rename a field between minor versions. This is a breaking change.

Reply:
Alright. Shall I open up a new PR for this (and similar) changes?
Those options just got introduced in the last patch, so changing it will not have a huge impact I'd say. But I respect your guidelines.
Still to be consistent (because other options and methods use collation with `collation` instead of `collationOptions`) I'd like to rename it anyhow. :)

Reply:
I had not seen the changes required in `MongoClient` when I wrote this comment. Given we're going to break the API anyway, I believe it is ok to make this change too.

Reply:
Thanks. next time I'll focus more on consistency in the first place.